palo_alto_panorama/assets/logs/palo-alto-panorama.yaml

```diff
@@ -0,0 +1,1255 @@
+          12:30:15|elapsed=0|tunnel_insp_rule=insepction_rule_1|remote_user_ip=10.1.2.3|remote_user_id=40201234567|rule_uuid=731925f6-8e55-xxyy-b82f-910b26004dfd|pcap_id=1223xa155545|dynusergroup_name=agnet145|src_edl=10.0.1.2|dst_edl=10.0.3.4|high_res_timestamp=2024-01-16T14:15:34.518+05:30|nssai_sd=|nssai_sst=|pdu_session_id=1234758662|subcategory_of_app=encrypted-tunnel|category_of_app=business-systems|technology_of_app=browser-based|risk_of_app=1|characteristic_of_app=used-by-malware,able-to-transfer-file|container_of_app=google|is_saas_of_app=1|sanctioned_state_of_app=0|cluster_name=test-nw-cluster
+      grok:
+        supportRules: ""
+        matchRules: >
+          rule_panorama (<%{integer}>)?((%{date("MMM  d HH:mm:ss")}|%{date("MMM
+          dd HH:mm:ss")}) (%{notSpace})?) %{data::keyvalue("="," \"/:,(){};+",
+          "", "|")}
+    - type: grok-parser
+      name: Parsing 'time_generated' to 'timestamp' (epoch)
+      enabled: true
+      source: time_generated
+      samples:
+        - Jan 24 2024 10:30:11 GMT
+      grok:
+        supportRules: ""
+        matchRules: date_rule %{date("MMM dd yyyy HH:mm:ss z"):timestamp}
+    - type: date-remapper
+      name: Define `timestamp` as the official date of the log
+      enabled: true
+      sources:
+        - timestamp
+    - type: service-remapper
+      name: Define `type` as the official service of the log
+      enabled: true
+      sources:
+        - type
+    - type: pipeline
+      name: Map `severity` to `panorama_severity`
+      enabled: true
+      filter:
+        query: '@type:("THREAT" OR "START" OR "END" OR "SYSTEM" OR "CORRELATION") '
+      processors:
+        - type: attribute-remapper
+          name: Map `severity` to `palo.alto.panorama.severity`
+          enabled: true
+          sources:
+            - severity
+          sourceType: attribute
+          target: palo.alto.panorama.severity
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+    - type: pipeline
+      name: Map `act` to `palo.alto.panorama.action`
+      enabled: true
+      filter:
+        query: "@type:(START OR END OR DECRYPTION OR THREAT OR TRAFFIC) "
+      processors:
+        - type: attribute-remapper
+          name: Map `act` to `palo.alto.panorama.action`
+          enabled: true
+          sources:
+            - act
+          sourceType: attribute
+          target: palo.alto.panorama.action
+          targetType: attribute
+          preserveSource: false
+          overrideOnConflict: false
+    - type: pipeline
+      name: Processors for "Decryption" OR "Traffic" OR "Threat" OR "Tunnel
+        Inspection" log types
+      enabled: true
+      filter:
+        query: '@type:("DECRYPTION" OR "TRAFFIC" OR "THREAT" OR "START" OR "END") '
```

Comment:
It seems it is the same query as the previous pipeline:
query: "@type:(START OR END OR DECRYPTION OR THREAT OR TRAFFIC) "
query: '@type:("DECRYPTION" OR "TRAFFIC" OR "THREAT" OR "START" OR "END") '

Unless I am missing something, cannot we group them under the same pipeline